### PR TITLE
Add Meilisearch recipe

### PR DESCRIPTION
**Pull request by codedge**
| Q             | A
| ------------- | ---
| License       | MIT

This PR adds the recipe for the [meilisearch/meilisearch-symfony](https://github.com/meilisearch/meilisearch-symfony) bundle.

### CODE REVIEW

File: meilisearch/search-bundle/0.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "MeiliSearch\\Bundle\\MeiliSearchBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MEILISEARCH_URL": "http://127.0.0.1:7700",
+        "MEILISEARCH_API_KEY": "",
```
Comment (Nyholm) [inline meilisearch/search-bundle/0.2/manifest.json:10]: Make sure you are using these in your bundle's config too

**Review by Nyholm — CHANGES_REQUESTED** [review]

File: meilisearch/search-bundle/0.2/manifest.json (hunk 1 above)
Comment (codedge) [reply meilisearch/search-bundle/0.2/manifest.json:10]: Fixed.

**Comment by codedge** [comment]
I think this build checks need to be triggered again. The PR implementing the correct usage of env vars has been merged.

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
I cannot restart travis (it failed to restart). Please add a new commit to this branch: 

```
git commit --allow-empty  -m "Trigger CI"
git push
```

**Review by Nyholm — APPROVED** [review]
Im happy with this. 

Could you rebase your PR so the new CI jobs run (I cannot merge it atm)

**Comment by codedge** [comment]
I rebased against the current `master`. Does not seem like a goold result 😅

**Comment by codedge** [comment]
Closing this in favour of https://github.com/symfony/recipes-contrib/pull/1188

**Pull request closed without merging**